test/unit/test_common.py

```
@@ -685,3 +686,25 @@ def test_volume_name_generation(self, mock_model):
 
             result = populate_volume_from_image(mock_model, Mock(engine="docker"), "test.gguf")
             assert result == expected_volume
+
+
+class TestStoreDiskStats:
+    @patch("ramalama.common.shutil.disk_usage")
+    def test_store_disk_stats(self, mock_disk_usage):
+        mock_disk_usage.return_value = (1000, 200, 800)
+        assert store_disk_stats(Path("/tmp/store")) == {"total": 1000, "used": 200, "free": 800}
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/store"

(S108)